### ISSUE TITLE
Empty Objects Not Collapsed using `object-curly-spacing` Rule

### ISSUE THREAD

**Issue by ciosciaa** [ota-meshi/eslint-plugin-jsonc#482]
Empty objects are not being collapsed to `{}` when using the `object-curly-spacing` rule. This issue has been observed in Version `3.0.0` with the following rule configuration:

```js
"jsonc/object-curly-spacing": [
	“warn”, “never”,
	{
		arraysInObjects: false,
		objectsInObjects: false
	}
]
```

# Demonstration
Given the following JSON with the above rule configuration…

```json
{
	"succeeds": { "works": true },
	"fails": { }
}
```

…The JSON should emit warnings before each bracket pair for both the `succeeds` and `fails` objects. This is not the case for the `fails` object brackets.

# Reproduction
This issue can be recreated as follows:

1. Clone the following repo: https://github.com/ciosciaa/eslint-plugin-jsonc-empty-bug/.
2. Install node modules via `npm i`.
3. Open the `test.json` file.
4. Run ESLint via `npx eslint`.
5. Observe that no issues are reported for the `fails` object brackets.

**Comment by ota-meshi** [comment]
I'm not entirely sure what you're posting.
Please provide the code that reproduces it, and explain what you expect and what's actually happening. Also, please share your repository so we can reproduce the issue.
